docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -1102,19 +1102,19 @@ Returns the receipt of a transaction by transaction hash. Receipts for pending t
 
 ### eth_newFilter
 
-Creates a logs filter to log state changes. To poll for state changes, use [eth_getFilterChanges](#eth_getfilterchanges).
+Creates a log filter. To poll for logs, use [eth_getFilterChanges](#eth_getfilterchanges).
```

Comment:
Maybe something more explicit like “To poll for logs associated with this filter” would help understand that the method is not just to retrieve logs, but logs that are returned by this specific filter. It was not super clear to me at first read so...

Reply:
Updated